integration-testing/src/main/java/io/grpc/testing/integration/Util.java

```diff
@@ -119,4 +131,42 @@ public static void assertEquals(List<? extends MessageLite> expected,
       }
     }
   }
+
+  /**
+   * Configure an OkHttpClientChannelBuilder so that it can use TLS correctly.
+   */
+  public static void configureTlsForOkHttpChannel(OkHttpChannelBuilder channelBuilder,
```

Comment:
This method seems to do a couple of different things. Could it simply accept a non-null certFile and return the SSLSocketFactory?